index.html

```diff
@@ -84,8 +88,7 @@ <h2 class="medium-title">Featured Categories</h2>
                 <a href="quiz-types.html">
                     <div class="card card-vertical category__card">
                         <div class="card__img">
```

Comment:
can use webp instead of jpg or png for

- Better Performance
- Compressed Size